scripts/src/public/services/offer_contents.ts

```diff
@@ -13,19 +13,32 @@ export enum PageType {
 	"EarnThankYou",
 }
 
+export interface BaseEarnPage {
+	type: PageType;
+	title: string;
+}
+
+export interface PollPage extends BaseEarnPage {
+	description: string;
+	question: Question;
+}
+
 export interface Poll {
-	pages: Array<{
-		type: PageType;
-		title: string;
-		description: string;
-		question: Question;
-	}>;
+	pages: Array<PollPage | EarnThankYouPage>;
```

Comment:
if you're referencing `EarnThankYouPage`, maybe move the definition to be above this one?